sql/clients_last28_v1.sql

```diff
@@ -0,0 +1,42 @@
+WITH
+  current_sample AS (
+  SELECT
+    * EXCEPT (submission_date_s3),
+    0 AS days_since_seen,
+    -- Active User means a client who has total_daily_uri >= 5
+    -- https://docs.telemetry.mozilla.org/cookbooks/active_dau.html
+    IF(scalar_parent_browser_engagement_total_uri_count_sum >= 5,
+      0,
+      NULL) AS days_since_active_user
```

Comment:
I don't think "active_user" is specific enough for this field name; it should specify 5 uri in the name somehow. Possibilities:
  - days_since_visited_5_uri
  - days_since_visited_5_or_more_uri
  - days_since_5_uri_active

Reply:
I figured linking to the `Active DAU` documentation that defines `Active User` at the very top would be enough to use `active_user`.

of those I like `days_since_visited_5_uri` the best

Reply:
My concern is that the "active_user" term is going to bleed to further stages of ETL, and I'd like to reduce the potential questions in a user's mind when looking at the schema. I think this is a case where we can make the meaning obvious without a user tracking down the ETL or the docs.